### PR TITLE
(SIMP-3632) Working InSpec acceptance test

### PR DESCRIPTION
**Pull request by trevor-vaughan**
* The test can be activated using 'rake beaker:suites[compliance]'
* Currently relies on the HEAD of puppetlabs_spec_helper
* Fully activates and uses the compliance_markup enforcement
* Only STIG checks activated for now

SIMP-3703 #close
SIMP-3704 #close
SIMP-3705 #close
SIMP-3632 #comment STIG profile tests functional

### CODE REVIEW

File: spec/acceptance/nodesets/default.yml [hunk 1]
```diff
@@ -7,22 +7,30 @@ HOSTS:
...
-      - el6
-    platform:   el-6-x86_64
-    box:        centos/6
-    hypervisor: vagrant
-    yum_repos:
-      epel:
-        mirrorlist: 'https://mirrors.fedoraproject.org/metalink?repo=epel-6&arch=$basearch'
-        gpgkeys:
-          - https://getfedora.org/static/0608B895.txt
+# el6:
```
[the remark this answers is not in the document]
Comment (trevor-vaughan) [reply spec/acceptance/nodesets/default.yml]: No, I'll fix it.

**Comment by lnemsick-simp** [comment]
When tests run, puppet reports a system reboot is required because audit has been modified.  However, the test doesn't reboot before checking compliance.  Is the compliance test valid if the reboot hasn't happened?

**Comment by lnemsick-simp** [comment]
Is this a problem?
el7 17:20:23$ inspec exec --format json /tmp/inspec/inspec_profiles/CentOS-7-disa_stig > /tmp/inspec_results.json
  WARN: Unresolved specs during Gem::Specification.reset:
        rake (>= 0)
  WARN: Clearing out unresolved specs.
  Please report a bug if this causes problems.

**Comment by trevor-vaughan** [comment]
@lnemsick-simp The first should be fixed, the second is just InSpec being weird but it's OK.

**Pull request merged**